### DISCUSSION TITLE
Licensing question

### DISCUSSION THREAD

**Discussion by PippoApps** [airsdk/Adobe-Runtime-Support#904] [General]
Just making sure.
The revenue CAP for the license, is related to the developer buying it, or the final client who buys the developer's services?

I.e. 
- Freelance Developer B has bought an AIR SDK license according to his personal revenue cap.
- Client A buys development services from Developer B. 
- Developer B delivers an app to be published on Client A Apple and Google stores, with Developer B's license. 

Is this correct?
Cheers

**Comment by ajwfrost** [comment]
Yes, it would just be up to Developer B to have the appropriate license based on their own company revenue. Doesn't matter what you then do with the application or who you're developing it on behalf of (which is one of the reasons we ended up with this model).

**Reply by PippoApps** [reply]
Thank you Andrew.

**Reply by marchbold** [reply]
Just FYI, we have taken a different approach and asked our clients to purchase licenses directly, we then use those licenses to develop their applications.

**Reply by ajwfrost** [reply]
Yes that's an alternative that would work

**Reply by Tim1973** [reply]
Can I ask, regarding pricing, Is revenue in the pricing plan based on the revenue made on app production/sales or on the total revenue made by the company?

**Reply by ajwfrost** [reply]
It's overall corporate revenue. See the "details" part of https://airsdk.harman.com/pricing:
> Revenue caps are based on your or your company’s turnover for the preceding 12 month accounting period.

**Reply by Tim1973** [reply]
Thanks for the reply Andrew. That doesn't seem to make sense as the app we produced only had a small audience and the revenue was very low.

**Reply by ajwfrost** [reply]
It may not be ideal in all cases but there are often multinational companies creating mobile apps that are given away for free e.g. as companion apps for their hardware products. So the application revenue would be zero for that scenario despite this being something of significant value to a company that could afford it...

thanks

**Reply by Tim1973** [reply]
Yes I see that now. Thanks for your time.
